### ISSUE TITLE
optimizer not working

### ISSUE THREAD

**Issue by hentienjoyer** [anthonyshook/dfsOptimizer#15]
hello I recently came across your optimizer and I don't know if I am doing it wrong or not but the optimizer return this error for NBA FD and DK "Error in build_lineups(opt, num_lineups = 5, verbose = FALSE) : Model could not reach a solution."  

code: `opt <- create_optimizer(site = 'Draftkings', 
                          sport = 'Basketball', 
                          contest_type = 'CLASSIC')
opt <- add_players_from_df(object = opt, df = playersDF)

# Generate lineups
lineups <- build_lineups(opt, num_lineups = 5, verbose = FALSE)`

[playersDF.csv](https://github.com/anthonyshook/dfsOptimizer/files/8257385/playersDF.csv)

Thanks.

**Comment by anthonyshook** [comment]
I'm unable to reproduce your error with the data set you have provided.  This code works without error for me:
```r
opt <- create_optimizer(site = 'Draftkings',
                          sport = 'Basketball',
                          contest_type = 'CLASSIC')
DF <- read.csv('./playersDF.csv')

# I have to remove the "Opponent" Column because that's not an available slot in the Player class
DF$Opponent <- NULL

# Add the players from the DF
opt <- add_players_from_df(object = opt, df = DF)
lineups <- build_lineups(opt, num_lineups = 5, verbose = FALSE)
```

**Comment by NahomTayeGH** [comment]
Hello, I am also getting similar error on a different df. I have not used the optimizer in a while and this is my first time seeing such error saying,  "Error in build_lineups(opt, num_lineups = 5, verbose = FALSE) : Model could not reach a solution." I have tried the code u provided on multiple computers and same error persists. I've also tried going back to R 4.0.2 and still get the same error. OP hasn't closed the post so i am assuming they also have not found a solution.
```
opt <- create_optimizer(site = 'Draftkings',
                          sport = 'Basketball',
                          contest_type = 'CLASSIC')

DF$id<-as.character(DF$id)
# Add the players from the DF
opt <- add_players_from_df(object = opt, df = DF)
lineups <- build_lineups(opt, num_lineups = 5, verbose = FALSE)
``` 
[DKNBA.csv](https://github.com/anthonyshook/dfsOptimizer/files/10279548/DKNBA.csv)

**Comment by anthonyshook** [comment]
Okay, I think I have figured out the issue:

For transparency, here's what happened -- when `ompr` version 0.8.0 returned a successful lineup, it would have a `status = 'optimal'` which we checked for.  In v 1.0.0, that status was changed to read `success`.

So the solution is to either downgrade `ompr` and `ompr.roi` to 0.8.0, or re-install the latest version of the dfsOptimizer app (1.4.2) from the main branch in this repo.

This has been tested on a Mac running R4.1, a Windows10 machine running R.4.0.2, and an Ubuntu server running R3.6
